src/dog.sol

```diff
@@ -151,88 +151,34 @@ contract Dog {
-    //
-    //     tab <= min(ilk.hole - ilk.dirt, Hole - Dirt)
-    //
-    // The debt of a vault is expressed in normalized terms:
-    //
-    //     vault.debt = vault.art * ilk.rate
-    //
-    // Furthermore, a vault's normalized debt is decreased after a liquidation
-    // is triggered:
-    //
-    //     vault.art' = vault.art - dart
-    //
-    // However, the new debt value cannot be dusty. This means that it has to be
-    // either zero, or at least equal to the collateral's `dust` value:
-    //
-    //     vault.art' * ilk.rate >= ilk.dust                                , or
-    //     vault.art' = 0
-    //
-    // Solving for `dart` for both cases using the previous equation,
-    //
-    //     vault.art' >= ilk.dust / ilk.rate
-    //     vault.art - dart >= ilk.dust / ilk.rate
-    //     -dart >= ilk.dust / ilk.rate - vault.art
-    //     dart <= vault.art - ilk.dust / ilk.rate
-    //
-    //     vault.art - dart = 0
-    //     dart = vault.art
-    //
-    // When `dart`'s value is one such that `vault.art'` will be dusty, the
-    // whole vault is liquidated instead, even if it surpasses the hole
-    // restriccions. This would happen for the following values of `dart`:
-    //
-    //     vault.art' * ilk.rate < ilk.dust && vault.art' > 0
-    //     vault.art' < ilk.dust / ilk.rate && vault.art' > 0
-    //     0 < vault.art' < ilk.dust / ilk.rate
-    //     0 < vault.art - dart < ilk.dust / ilk.rate
-    //     - vault.art < -dart < ilk.dust / ilk.rate - vault.art
-    //     vault.art > dart > vault.art - ilk.dust / ilk.rate
-    //     vault.art - ilk.dust / ilk.rate < dart < vault.art
-    //
-    // Thus, the general equation for `tab` is as follows:
-    //
-    //     tab = min(
-    //             dart * ilk.rate * ilk.chop,
-    //             ilk.hole - ilk.dirt,
-    //             Hole - Dirt
-    //           )                 , for dart <= vault.art - ilk.dust / ilk.rate
-    //
-    //     tab = vault.art * ilk.rate * ilk.chop
-    //                 , for vault.art - ilk.dust / ilk.rate < dart <= vault.art
+    // an auction and `lot`, the collateral to put for sale.
+    // They are defined as:
+    //
+    // tab = dart * ilk.rate * ilk.chop
+    // lot = dink = urn.ink * dart / urn.art
+    //
+    // `room` is defined as the remaining space of
+    // available DAI to be offered in auctions:
+    // room = min(Hole - Dirt, ilk.hole - ilk.dust)
+    //
+    // There is a precondition about `room` that needs
+    // to be satisfied in order to create an auction:
+    // room > 0 && room >= ilk.dust
+    // otherwise the transaction fails
+    //
+    // Then `dart` is defined as:
+    // if urn.art <= room || urn.art > room && urn.art < room + dust
+    //      dart = urn.art
+    // otherwise
+    //      dart = room / ilk.rate / ilk.chop
+    //
+    // Leaving the final formulas as:
+    // if urn.art <= room || urn.art > room && urn.art < room + dust:
```

Comment:
`urn.art * ilk.rate` in order for it to be comparable to `room`

Reply:
It should have the `* ilk.chop` as well. Pushed a commit for it.